web/client/components/widgets/enhancers/legendWidget.js

```diff
@@ -18,8 +18,8 @@ const {getScales} = require('../../../utils/MapUtils');
  * Add also base tools and menu to the widget
  */
 module.exports = compose(
-    withProps(({ dependencies = {} }) => ({
-        layers: dependencies.layers || [],
+    withProps(({ dependencies = {}, dependenciesMap = [] }) => ({
```

Comment:
```suggestion
    withProps(({ dependencies = {}, dependenciesMap = {} }) => ({
```
By default is an object